test/widgets/message_list_test.dart

```diff
@@ -101,15 +101,17 @@ void main() {
     testWidgets('MessageListPageState.narrow', (tester) async {
       final stream = eg.stream();
       await setupMessageListPage(tester, narrow: ChannelNarrow(stream.streamId),
-        messages: [eg.streamMessage(stream: stream, content: "<p>a message</p>")]);
+        messages: [eg.streamMessage(stream: stream, content: "<p>a message</p>")],
+        streams: [stream]);
```

Comment:
nit:
```suggestion
        streams: [stream],
        messages: [eg.streamMessage(stream: stream, content: "<p>a message</p>")]);
```

The message lives within the stream, so this is the logical order. Also `messages` is the main "payload" parameter, while the others are more like metadata — much like `child` or `children` vs. other parameters in a typical Flutter widget constructor.